types/hapi/definitions/server/server-events.d.ts

```diff
@@ -123,7 +174,7 @@ export interface ServerEvents extends Podium {
      * @return Return value: a promise that resolves when the event is emitted.
      * [See docs](https://github.com/hapijs/hapi/blob/master/API.md#-await-servereventsoncecriteria)
      */
-    once(criteria: string | ServerEventsApplicationObject | ServerEventCriteria): Promise<any>;
+    once(criteria: string | ServerEventCriteria<string>): Promise<any>;
```

Comment:
Aren't these the same as above?

Also you could get a little bit _smart_ to avoid writing the same code over and over again, which saves code (tradeoff is weird autocompletion is VSCode).

https://github.com/SimonSchick/EliteDangerousUtils/blob/master/src/EDLog/EDLog.ts#L307

On the nice side you can use this as a generic interface and declare all methods at once!